Public/Src/Engine/Scheduler/Tracing/Log.cs

```diff
@@ -609,6 +609,15 @@ internal static string PipWriteFileFailedMessage(string pipDescription, string p
             Message = "Cleaning output directory '{destinationPath}' for pip {pipDescription} resulted in error: {errorMessage}")]
         public abstract void StorageCacheCleanDirectoryOutputError(LoggingContext loggingContext, string pipDescription, string destinationPath, string errorMessage);
 
+        [GeneratedEvent(
+            (ushort)LogEventId.StorageSymlinkDirInOutputDirectoryWarning,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Warning,
+            Keywords = (int)Events.Keywords.UserMessage,
+            EventTask = (ushort)Events.Tasks.Storage,
+            Message = "[{pipDescription}] Cache pip with symlink directory '{symlinkPath}' in output directory resulted in error")]

```

Comment:
I wouldn't say "resulted in **error**" when this is a warning message.

Instead, I'd say something like "Detected junctions or directory symlink output '{symlinkPath}'. The pip will not be stored to cache."

Reply:
Thanks